google/cloud/aiplatform/training_jobs.py

```diff
@@ -386,12 +417,47 @@ def _create_input_data_config(
 
         input_data_config = None
         if dataset:
-            # Create fraction split spec
-            fraction_split = gca_training_pipeline.FractionSplit(
-                training_fraction=training_fraction_split,
-                validation_fraction=validation_fraction_split,
-                test_fraction=test_fraction_split,
-            )
+            filter_split = None
+            if any(
+                [
+                    training_filter_split is not None,
+                    validation_filter_split is not None,
+                    test_filter_split is not None,
+                ]
+            ):
+                if all(
+                    [
+                        training_filter_split is not None,
+                        validation_filter_split is not None,
+                        test_filter_split is not None,
+                        dataset._gca_resource.metadata_schema_uri
+                        in (
+                            schema.dataset.metadata.image,
```

Comment:
This should be a constant at the top of this file. Something like `_DATASETS_WITH_DATA_ITEMS`.